samples/videoDecode/README.md

```diff
@@ -38,4 +38,5 @@ make -j
               -md5_check MD5_File_Path <generate MD5 message digest on the decoded YUV image sequence and compare to the reference MD5 string in a file [optional]>
               -crop <crop rectangle for output (not used when using interopped decoded frame) [optional - default: 0,0,0,0]>
               -m <output_surface_memory_type - decoded surface memory [optional - default: 0][0 : OUT_SURFACE_MEM_DEV_INTERNAL/ 1 : OUT_SURFACE_MEM_DEV_COPIED/ 2 : OUT_SURFACE_MEM_HOST_COPIED/3 : OUT_SURFACE_MEM_NOT_MAPPED]>
+              -no_ffmpeg_demux <use the built-in bitstream reader instead of FFMPEG demuxer to obtain picture data [optional]>
```

Comment:
does this option handle all the streams. Otherwise, we need not add this option here as we have a separate sample

Reply:
We need to add the bitstream reader option to the general decode sample to use MD5, for conformance tests. The videodecoderaw sample does not use any FFMPEG utilities.

Reply:
Done.